study.md

```diff
@@ -67,5 +68,4 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
-```
+Client submits GET request to server -> server translates GET request and sends to controller -> controller submits GET to model -> model performs request and returns result to controller -> controller sends GET request to server -> server translates GET request and sends to client.
```

Comment:
This is correct except for "controller submits GET to model" and "controller sends GET request to server -> server translates GET request and sends to client.".  The controller renders the response to the client.